AllReadyApp/Web-App/AllReady/web.config

```diff
@@ -4,6 +4,6 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
     </handlers>
-    <aspNetCore processPath=".\AllReady.exe" arguments="" forwardWindowsAuthToken="false" stdoutLogEnabled="false" />
+    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" forwardWindowsAuthToken="false" stdoutLogEnabled="false" />
```

Comment:
Please revert the changes to this file. Until we figure out the environment variable situation for our Azure deployments, we need to hardcode `processPath` and `arguments`.